karma.conf.js

```diff
@@ -7,7 +7,7 @@
 module.exports = function (config) {
   config.set({
     files: [
-      'test/test.js'
+      { pattern: 'test/**/*.spec.js' }
```

Comment:
I have this refactor in another PR which I will submit separately then rebase this branch.

Reply:
Sure, makes sense. Let's merge my PR for tests first then: #23